pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -45,6 +41,10 @@ import (
 	"knative.dev/serving/pkg/metrics"
 	"knative.dev/serving/pkg/network"
 	"knative.dev/serving/pkg/resources"
+	tracingconfig "knative.dev/serving/pkg/tracing/config"
+	"sort"
+	"strconv"
+	"testing"
```

Comment:
Please run `goimports` on your source. It will `gofmt` and sort the imports in "the right way" (tm)

Reply:
thanks - was using `gofmt`, but it looks like `goimports` is a superset of `gofmt` also ordering imports correctly.